scripts/error-codes/codes.json

```diff
@@ -143,5 +143,46 @@
   "141": "Expected onSetChildren() to fire for a container child before its parent includes it in onSetChildren().",
   "142": "Expected onBeforeMountComponent() parent and onSetChildren() to be consistent (%s has parents %s and %s).",
   "143": "React.Children.only expected to receive a single React element child.",
-  "144": "React.PropTypes type checking code is stripped in production."
-}
\ No newline at end of file
+  "144": "React.PropTypes type checking code is stripped in production.",
```

Comment:
Oh I see now, it's not testing this specific invariant it's just testing _any_ production invariant. That makes sense, we should definitely change it.